game.py

```diff
@@ -51,18 +52,27 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, known=True, **kwargs)
 
 
-def lookup_game_by_name_or_alias(name):
-    # Name may contain extra junk, e.g. "I'd play cs later, after food" would mean name="cs later, after food"
-    for game in games:
+def lookup_known_game_by_name_or_alias(name):
```

Comment:
This is fine, and I remember adding it, but I could have sworn that was through Dan's PR. Am I going crazy? Was this definitely part of your changes?